README.md

```diff
@@ -1,2 +1,35 @@
 # 1st-Capstone
-This project is aimed testing the skills of html and css at Microverse
\ No newline at end of file
+This  project is aimed at building a beautiful landing page.
+
+# Built With
+HTML
+CSS
+
+
+# live Demo
+ To see a live demo kindly visit [Capstone](https://amklass11.github.io/1st-Capstone/)
+
+# Getting Started
+This is a  microverse project aimed at grasping responsive design wtih html
+and css by building the 2 pages of a website. 
+
+# Authors
+Amk lass
+
+GitHub:[@amklass11](https://github.com/amklass11)
+LinkedIn:[@AmkLAss](https://www.linkedin.com/in/amk-lass-521565196/)
+# 🤝 Contributing
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+# Show your support
+Give a ⭐️ if you like this project! and cheers✨
+
+ # Acknowledgments
+Microverse offered guidance on how to make the project more professional
```

Comment:
- [ ] The README contains an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).